libqtile/window.py

```diff
@@ -785,6 +811,7 @@ def __init__(self, window, qtile):
             win = qtile.windows_map.get(transient_for)
             if win is not None:
                 group = win._group
+                self.z = (win.z or NORMAL_LAYER) + 1  # stay above parent
```

Comment:
Why the `or`? `win.z` will always return an integer value, right?

Reply:
Yes, it will be an integer. The user _could_ set it to `None`, though. Should I rather expect a reasonable user and simplify the line?

Reply:
Yeah let's keep it simple. I wonder if we really need the +1 also, I think the X server should handle that for us by default.